### ISSUE TITLE
expose GenerateForecastMetricsRequest

### ISSUE THREAD

**Issue by jafaircl** [Opteo/google-ads-api#71]
### I'm submitting a ...

-   [ ] bug report
-   [x] feature request
-   [ ] question about the decisions made in the repository
-   [ ] question about how to use this project

### Summary

Is there a timeline for exposing GenerateForecastMetricsRequest for KeywordPlans in the API? I can see that it's [generated in google-ads-node](https://github.com/Opteo/google-ads-node/blob/master/src/protos/google/ads/googleads/v2/services/keyword_plan_service_pb.d.ts) so I'm guessing it's possible to use it straight from that library. But, it would be nice if it would be exposed here as well. I'm happy to help out if anyone can point to an example of how to get a service from google-ads-node into this library.

### Other information

N/A

**Comment by jafaircl** [comment]
Adding this method to KeywordPlanService seems to work but is it the way y'all would want it to be done?

```typescript
import { GenerateForecastMetricsRequest, GenerateForecastMetricsResponse } from 'google-ads-node'

export default class KeywordPlanService extends Service {

    ...

    public async generateForecastMetrics(id: string) {
        const request = new GenerateForecastMetricsRequest()
        request.setKeywordPlan(`customers/${this.cid}/${RESOURCE_URL_NAME}/${id}`)
        return (await this.service.generateForecastMetrics(request)) as GenerateForecastMetricsResponse
    }
}
```

**Comment by kritzware** [comment]
Thanks for the PR! This should now be available in [v3.4.0](https://www.npmjs.com/package/google-ads-api) 👍
